### DISCUSSION TITLE
Properly Using Greenlight on Mac

### DISCUSSION THREAD

**Discussion by samgocubsgo** [unknownskl/xbox-xcloud-client#797] [Q&A]
Hey all, this is a dumb Q but having trouble across versions on my MacBook Probs. I downloaded just the .dmg but finding the stream to be a bit laggy. Should i be downloading any other files that would help this? Or is this just a MacBook Pro issue?

Thanks in advance, love the software and idea

**Comment by unknownskl** [comment]
It should work fine without any other software. If you are using v1.2.0 then try to upgrade to v2.0.0-beta6. It has a lot of updates which should improve the latency.
If the stream is still laggy then it might be the way how the xbox and macbook is connected to the network. The best performance would be to have both devices connected by ethernet.
